challenges/objects-arrays.js

```diff
@@ -85,35 +128,53 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 
 /* Request 1: .forEach()
 
-The zoo wants to display both the scientific name and the animal name in front of the habitats.  Return an array with only the animal and scientific names in it.  The individual values in the array should look like this "Name: Jackal, asiatic, Scientific: Canis aureus."
+The zoo wants to display both the scientific name and the animal name in front of the habitats.  Return an array with only the animal and scientific names in it.  
+The individual values in the array should look like this "Name: Jackal, asiatic, Scientific: Canis aureus."
 
 */
 const animalNames = [];
+
+zooAnimals.forEach(element => {
+  animalNames.push(`${element.animal_name}, ${element.scientific_name}`)
+});
+
 console.log(animalNames);
 
 /* Request 2: .map()    
 
-The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
+The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and 
+map over each name to convert them all to lower case.  Log the resut.
 
 */
 
-const lowerCase = [];
+// const lowerCase = [];
+
+const lowerCase = zooAnimals.map(element => element.animal_name.toLowerCase())
+
 console.log(lowerCase); 
 
 /* Request 3: .filter() 
 
 The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
 
 */
-const largerPopulation = [];
-console.log(largerPopulation);
+const lowerPopulation = zooAnimals.filter(element => element.population < 5);
+
+console.log(lowerPopulation);
 
 /* Request 4: .reduce() 
 
 The zoos need to know their total animal population across the United States.  Find the total population from all the zoos using the .reduce() method.
 
 */
-const populationTotal = 0;
+// const populationTotal = 0;
+
+function adder(acc, element) {
+  return acc + element.population;
+}
+
+const populationTotal = zooAnimals.reduce(adder, 0)
+
 console.log(populationTotal);
 
```

Comment:
Great work on using your loops. Especially properly using each method. You have a good understanding of each one.